src/routes/(modeles-services)/_common/fields-modalities-coach.svelte

```diff
@@ -17,18 +17,25 @@
   export let servicesOptions: ServicesOptions;
 
   let coachOrientationModesFocusValue: string | undefined = undefined;
+  let preventUpdateExternalFormFields = true;
 
-  $: {
-    // Suppression du lien vers le formulaire externe et de son intitulé
-    // lorsque la case du formulaire externe est décochée.
-    if (
-      !service.coachOrientationModes.includes(
-        "completer-le-formulaire-dadhesion"
-      )
-    ) {
-      service.coachOrientationModesExternalFormLink = "";
-      service.coachOrientationModesExternalFormLinkText = "";
+  function updateExternalFormFields(text: string) {
+    if (preventUpdateExternalFormFields) {
+      // Don't reset fields on first load
```

Comment:
_nit_ : In french in the text ?

Reply:
Corrigé dans 73d00c504f7631ef965fbb7c07e9819df7d841d3